README.md

```diff
@@ -48,6 +48,28 @@ Development
 	bundle
 	foreman start
 
+Setup
```

Comment:
Would you please rearrange and rename the steps for development setup as follows:

    ## Development Setup
    ### Without Docker
    ### With Docker